### PR TITLE
fix 1356: diet for composer

### PR DESCRIPTION
**Pull request by nicosomb**
I tried locally (composer clear cache, rm vendor -rf, composer install): no more "cloning ...". Only downloading.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -69,7 +49,6 @@
         "sensio/framework-extra-bundle": "~3.0",
         "incenteev/composer-parameter-handler": "~2.0",
         "nelmio/cors-bundle": "~1.4.0",
-        "umpirsky/twig-gettext-extractor": "~1.1.5",
```
Comment (nicosomb) [inline composer.json]: already embedded with an other dependancy

File: composer.json [hunk 2]
```diff
@@ -29,30 +29,10 @@
...
-            "type": "vcs",
-            "url": "https://github.com/wallabag/pagination"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/wallabag/PHPePub"
-        },
-        {
-            "type": "vcs",
             "url": "https://github.com/wallabag/php-readability"
```
Comment (j0k3r) [inline composer.json]: This one is still required ?

Comment (nicosomb) [inline composer.json]: yes, we still use Readability class.

Comment (j0k3r) [inline composer.json]: Oh sorry I was thinking that removing all lines and just add tag to require will be enough, my bad.

**Pull request merged**